server.py

```diff
@@ -59,6 +64,6 @@ def do_POST(self):
             SimpleHTTPRequestHandler.do_POST(self)
 
 if __name__ == '__main__':
-    print "Server started: http://localhost:3000/"
+    print("Server started: http://localhost:3000/")
```

Comment:
This works because there is only one string, so it's equivalent to not putting any parenthesis. Nevertheless, it does not magically transforms the print keyword into a function. Try adding another string and see the monster you just created! If you want the real Py3 `print()` function, you have to add this as the first line:

```python
from __future__ import unicode_literals, print_function
```